src/app/campaign.service.ts

```diff
@@ -44,6 +47,23 @@ export class CampaignService {
     return 100 * campaign.amountRaised / campaign.target;
   }
 
+  buildQuery(selected: SelectedType, offset: number): {[key: string]: any} {
+    const perPage = 6;
+    const query: SearchQuery = {
```

Comment:
@NoelLH the meta campaign pages are including child campaigns that aren't associated to the master, I think because `parentCampaignId` here isn't set when loading the meta pages.